engine/runtime/src/main/scala/org/enso/compiler/core/IR.scala

```diff
@@ -736,6 +736,9 @@ object IR {
             s"""
             |IR.Module.Scope.Import.Module(
             |name = $name,
+            |rename = $rename,
+            |onlyNames = $onlyNames,
+            |hiddenNames = $hiddenNames,
```

Comment:
`isAll` is missing in the dump. Consider picking https://github.com/enso-org/enso/pull/3611/commits/34f49f299ca1ac2b3604c486338a3f6a1ffe9cca up.